spinner/spinner_other.go

```diff
@@ -0,0 +1,13 @@
+// +build dragonfly freebsd linux netbsd openbsd solaris darwin
```

Comment:
You forgot Plan 9 ;-)
How about `// +build !windows`?

Also, can we name this file `spinner_unix.go`? I think it's clearer.